rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -2427,12 +2427,12 @@ def test_upgrade_db_31_to_32(user_data_dir):  # pylint: disable=unused-argument
     cursor = db_v31.conn.cursor()
     result = cursor.execute('SELECT rowid from history_events')
     old_ids = {row[0] for row in result}
-    assert len(old_ids) == 3
+    assert len(old_ids) == 13
     cursor.execute(
         'SELECT subtype from history_events',
     )
     subtypes = [row[0] for row in cursor]
-    assert set(subtypes) == {'staking deposit asset', 'staking receive asset', None}
+    assert set(subtypes) == {'staking deposit asset', 'staking receive asset', None, 'fee'}
```

Comment:
I would like to also see a test case here for what I mentioned for the DB upgrade and the duplicates existing in the DB before the upgrade and being "fixed" after.